ros/resources/v0/hosts.py

```diff
@@ -43,20 +64,37 @@ def get(self):
 
         inv_hosts = fetch_all_hosts_from_inventory(auth_key)
         inv_host_ids = [host['id'] for host in inv_hosts['results']]
-        profile_result = PerformanceProfile.query.filter(
+
+        # Note that When using LIMIT, it is important to use an ORDER BY clause
+        # that constrains the result rows into a unique order.
+        # Otherwise you will get an unpredictable subset of the query's rows.
+        # Refer - https://www.postgresql.org/docs/13/queries-limit.html
+        query = PerformanceProfile.query.filter(
             PerformanceProfile.inventory_id.in_(inv_host_ids)).order_by(
-                PerformanceProfile.report_date.desc()).all()
+                PerformanceProfile.report_date.desc()).order_by(
+                    PerformanceProfile.id.asc())
```

Comment:
Adding the `asc()` order_by clause will change the ordering/result and it can break the below logic of getting distinct hosts with latest report_date.

Reply:
@patilsuraj767, @apuntamb,
Yes! Currently I have applied order_by `id`  because I was getting an unpredictable rows & few rows were missing from DB.
`report_date` values can be same for records so not sure if that gives unique order.

As per my understanding, fetching of unique host_id's latest record logic should be handled at query level not here ->
https://github.com/RedHatInsights/ros-backend/pull/18/files#diff-e303aa4e3e61a69895b0d295ce9166a08965a63a0c63569a4e1c8c60235f3b50R82

I have reported a bug  - https://issues.redhat.com/browse/RHIROS-59. 
This change should not be a part of pagination change.

Reply:
I have marked issue ticket as bug because currently I applied order by `id` but for sorting change, this implementation needs to be fixed.